sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -183,6 +187,17 @@ public AzureMonitorExporterBuilder serviceVersion(AzureMonitorExporterServiceVer
         return this;
     }
 
+    /**
+     * Sets the token credential required for authentication with the ingestion endpoint service.
+     *
+     * @param credential The Azure Identity TokenCredential.
+     * @return The updated {@link AzureMonitorExporterBuilder} object.
+     */
+    public AzureMonitorExporterBuilder credential(TokenCredential credential) {
```

Comment:
This is not an alternative for connectionString right? This would still require setting the connection string?

Reply:
For the SDK , we dont want to take dependency on connection string. The agent will build the tokenCredential and use it while initialization.